README.md

```diff
@@ -163,6 +162,45 @@ The user needs to reset their password to continue with the authentication flow
 idxAuthenticationWrapper.revokeToken(TokenType.ACCESS_TOKEN, accessToken);
 ```
 
+### Register a User
+
+```java
+// begin transaction
+AuthenticationResponse beginResponse = idxAuthenticationWrapper.begin();
+
+// get proceed context
+ProceedContext beginProceedContext = beginResponse.getProceedContext();
+
+// set user profile
+UserProfile userProfile = new UserProfile();
+userProfile.addAttribute("lastName", lastname);
+userProfile.addAttribute("firstName", firstname);
+userProfile.addAttribute("email", email);
+
+# register user with proceed context context
+AuthenticationResponse authenticationResponse =
+        idxAuthenticationWrapper.register(proceedContext, userProfile);
```

Comment:
Will this work? I thought you needed another SDK call between this and begin

Reply:
good catch, updated!